tools/ogstools/msh2vtu.xml

```diff
@@ -0,0 +1,97 @@
+
+<tool id="msh2vtu" name="ogstools msh2vtu" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>
+        Convert msh to vtu/vtk.
+    </description>
+    <macros>
+        <import>macros.xml</import>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <expand macro="ogsTools_requirements"/>
+    
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input_file' input_file.msh &&
+        msh2vtu input_file.msh 
+            -d '$dim' 
+            $delz $swapxy $reindex $keep_ids $ascii
+            -l '$log_level'
+    ]]></command>
+    <inputs>
+        <param name="input_file" type="data" format="GMSH" label="Input GMSH File"/>
+        <param name="dim" type="select" label="Mesh Dimension" argument="-d">
```

Comment:
<param name="dim" type="select" label="Mesh Dimension" argument="-d">
Would change label to: "Set dimensionality of the mesh"